README.md

```diff
@@ -9,6 +9,8 @@ This suite tests compliance of a promise implementation with the [Promises/A+ sp
 
 [Promises/A+ specification]: https://github.com/promises-aplus/promises-spec
 
+Passing the tests in this repo means that you have a Promises/A+ compliant implementation of the `then()` method, and you can display the Promises/A+ logo in your README. You can also send a PR to have your impl listed on the [implementations page](https://promisesaplus.com/implementations).
```

Comment:
s/PR/pull request, and maybe link to the main repo.

s/impl/implementation

Otherwise LGTM!